coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentDocument.java

```diff
@@ -611,8 +611,8 @@ public NoteType getNoteType() {
 
     @Override
     public List getNotes() {
-        if (CollectionUtils.isEmpty(notes)) {
-            notes = new ArrayList<>(super.getNotes());
+        if (StringUtils.isNotBlank(getNoteTarget().getObjectId())) {
```

Comment:
this seems like a lot of overhead.  Is there a way we can do this only if the CollectionUtils.isEmpty(notes) as well?